components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/css/custom.css

```diff
@@ -148,7 +148,7 @@ button.ui-datepicker-current { display: none; }
     width: 100%;
 }
 
-#singleEventConfigTabContent>.tab-pane>.content {
+#single-event-config-tab-content>.tab-pane>.content {
     overflow-y: auto;
     position: relative;
     height: 84%;
```

Comment:
This is done to avoid content overlapping with the footer. Yes, it is responsive.